.github/workflows/tests.yml

```diff
@@ -22,12 +22,12 @@ on:
       - "requirements.txt"
 
 jobs:
-  python-tests:
+  security-check:
```

Comment:
MAYBE: I think we should create another job? (in this one there are also the tests)